runtime/src/accounts.rs

```diff
@@ -443,11 +444,9 @@ impl Accounts {
         if account_hashes.is_empty() {
             None
         } else {
-            let mut hasher = Hasher::default();
-            for (_, _, hash) in account_hashes {
-                hasher.hash(hash.as_ref());
-            }
-            Some(hasher.result())
+            let leaf_data: Vec<_> = account_hashes.iter().map(|ah| ah.2.as_ref()).collect();
```

Comment:
if you rebase, this gets prettier, I think ;)

Reply:
Sure does!  Just resolved the conflict :)